src/index.html

```diff
@@ -11,6 +11,29 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="search-form search-form--big" data-qa="big"
+    action="#" method="GET">
```

Comment:
code it not aligned, check the checklist
```suggestion
    <form 
       class="search-form search-form--big"
       data-qa="big"
       action="#" method="GET"
     >
```
![Uploading Screenshot 2023-09-23 at 14.52.14.png…]()